sw/airborne/boards/bebop/mt9f002.h

```diff
@@ -77,6 +77,8 @@ struct mt9f002_t {
   struct i2c_transaction i2c_trans;   ///< I2C transaction for comminication with CMOS chip
 };
 
+
+extern void set_offset(struct mt9f002_t *mt);
```

Comment:
doxygen description of what this function actually does would be nice...
Also would make sense to call it mt9f002_set_offset